seismicpro/src/weathering_velocity/metrics.py

```diff
@@ -0,0 +1,58 @@
+"""Implements the weathering velocity metrics"""
+
+import numpy as np
+
+from ..metrics import PipelineMetric, pass_calc_args
+from ..const import HDR_FIRST_BREAK
+
+
+class WeatheringVelocityMetric(PipelineMetric):
+    """Calculate a weathering metric. Weathering metric is fraction of first breaking times that stands out from
+    a weathering velocity curve (piecewise linear function) more than threshold.
+    """
+    name = "weathering_metrics"
+    vmin = 0
+    vmax = 0.05
+    is_lower_better = True
+    views = ("plot", "plot_wv")
+    args_to_unpack = ("gather", "weathering_velocity")
+
+    @staticmethod
+    def calc(gather, weathering_velocity, first_breaks_col=HDR_FIRST_BREAK, threshold_times=50, **kwargs):
```

Comment:
I'd suggest to rename `threshold_time` to `threshold_ms` - this way the units become obvious
```suggestion
    def calc(gather, weathering_velocity, first_breaks_col=HDR_FIRST_BREAK, threshold_ms=50, **kwargs):
```